### PR TITLE
Edit entries python codebyte

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- Updated the following entries:
  - Python Math functions: .ceil(),
  - Python Math functions: .cos(),
  - Python Math functions: .log(),
 
- With the following changes:
  - Add Codebyte section,
  - Fix typos.

- Closes: #2885 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/math-module/terms/math-ceil/math-ceil.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Takes in a numeric data type and rounds the value up to the nearest integer.
 math.ceil(n)
 ```
 
-The `math.ceil()` function takes in a value and returns its ceiling, the smallest integer greater than or equal to that value.
+The **`math.ceil()`** function takes in a value and returns its ceiling, the smallest integer greater than or equal to that value.
 
 ## Example 1
```
Comment (caupolicandiaz) [inline content/python/concepts/math-module/terms/math-ceil/math-ceil.md:25]:
```suggestion
## Example 
```

Comment (caupolicandiaz) [reply content/python/concepts/math-module/terms/math-ceil/math-ceil.md:25]: I would also suggest removing one of the existing examples from these entries. These are all generally pretty basic methods and there isn't a lot gained by showing essentially the same functionality multiple times.

File: content/python/concepts/math-module/terms/math-cos/math-cos.md [hunk 2]
```diff
@@ -12,15 +12,15 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-Returns the cosine of the angle.
+Returns the cosine of an angle.
 
 ## Syntax
 
 ```py
```
Comment (caupolicandiaz) [inline content/python/concepts/math-module/terms/math-cos/math-cos.md:19]:
```suggestion
```pseudo
```

Comment (caupolicandiaz) [reply content/python/concepts/math-module/terms/math-cos/math-cos.md:19]: update all entries accordingly

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cslylla I've completed my initial review. Please update the entries, and tag me when you've finished making changes. Thanks

**Comment by cslylla** [comment]
@caupolicandiaz Thank you for the comments. I updated all files accordingly. I checked the style guide, and all the other files in the math function folder are incorrect (in issue #2888, #2887, #2886):
- Syntaxy should be `pseudo` (not `py`)
- Example should be `py` (not `python`)
- Codebyte (if there) should be `codebyte/python` (not `codebyte/py`)

I also noticed that the `.ceil()` function from this issue is also included in issue #2888.

**Review by caupolicandiaz — APPROVED** [review]
@cslylla Thanks for going through and reviewing the other terms and highlighting the duplicated issue. This looks good, ready for a second review.

File: content/python/concepts/math-module/terms/math-cos/math-cos.md [hunk 3]
```diff
@@ -12,36 +12,34 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-Returns the cosine of the angle.
+Returns the cosine of an angle.
```
Comment (SSwiniarski) [inline content/python/concepts/math-module/terms/math-cos/math-cos.md:15]:
```suggestion
The **`math.cos()`** function returns the cosine of an angle.
```

File: content/python/concepts/math-module/terms/math-cos/math-cos.md [hunk 4]
```diff
@@ -12,36 +12,34 @@ CatalogContent:
...
 
 ## Syntax
 
-```py
+```pseudo
 math.cos(n)
 ```
 
-The `math.cos()` function returns a numeric value between -1 and 1, which represents the cosine of the value `n`.
+The **`math.cos()`** function returns a numeric value between -1 and 1, which represents the cosine of the value `n`.
```
Comment (SSwiniarski) [inline content/python/concepts/math-module/terms/math-cos/math-cos.md:23]:
```suggestion
The `math.cos()` function returns a numeric value between -1 and 1, which represents the cosine of the value `n`.
```

File: content/python/concepts/math-module/terms/math-log/math-log.md [hunk 5]
```diff
@@ -12,36 +12,40 @@ CatalogContent:
   - 'paths/computer-science'
 ---
 
-Returns the natural logarithm of `n`.
+Returns the natural logarithm of a number or the logarithm of a number to the given base.
```
Comment (SSwiniarski) [inline content/python/concepts/math-module/terms/math-log/math-log.md:15]:
```suggestion
The **`math.log()`** function returns the natural logarithm of a number or the logarithm of a number to the given base.
```

File: content/python/concepts/math-module/terms/math-log/math-log.md [hunk 6]
```diff
@@ -12,36 +12,40 @@ CatalogContent:
...
 
 ## Syntax
 
-```py
+```pseudo
 math.log(n, base)
 ```
 
-`base` is optional and it's the logarithmic base to use. Default is e.
+The **`math.log()`** function takes the following parameters:
```
Comment (SSwiniarski) [inline content/python/concepts/math-module/terms/math-log/math-log.md:23]:
```suggestion
The `math.log()` function takes the following parameters:
```

File: content/python/concepts/math-module/terms/math-log/math-log.md [hunk 7]
```diff
@@ -12,36 +12,40 @@ CatalogContent:
...
+- `n` is a required number or numeric expression to calculate the logarithm.
+- `base` is an optional number to specify the logarithm base. The default value for the base is `e`.
+
+The `math.log()` function returns a `float` value, the natural logarithm of `n` or the logarithm of `n` to `base`. If `n` is `0` or a negative number, it returns a `ValueError`.
 
-## Example 1
+## Example
 
-Use `math.log()` to return the natural log of `12` (base e):
+Use `math.log()` to return the natural log of `12` (base `e`):
```
Comment (SSwiniarski) [inline content/python/concepts/math-module/terms/math-log/math-log.md:32]:
```suggestion
Use `math.log()` to return the natural log of `12` (base _e_):
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@cslylla I did a second review and left some edits which were mostly updates to the original entries to make them conform more to current practice.

**Review by SSwiniarski — APPROVED** [review]
@cslylla Looks good to merge.

**Pull request merged**